app/models/rate.rb

```diff
@@ -0,0 +1,6 @@
+class Rate < ActiveRecord::Base
+  belongs_to :rater, class_name: 'User'
+  belongs_to :rateable, polymorphic: true
+
+  # attr_accessible :rate, :dimension
```

Comment:
I don't think we do. This is automatically generated code. I'll look up more for comments/things like these that can be safely removed. Although, I shouldn't change much except comments as I won't know reason things were introduced.